### ISSUE TITLE
Bug : I can't even build project for debugging

### ISSUE THREAD

**Issue by joukhar** [MakeOpinionGmbH/cpx-research-SDK-Flutter#2]
```
../../../.pub-cache/git/cpx-research-SDK-Flutter-aaaa7214bcb21594fc77658e2a98460e7bd7656a/cpx_research_sdk_flutter/lib/widgets/cpx_survey_cards.dart:142:13: Error: No named parameter with the name 'primary'.
            primary: config.cardBackgroundColor,
            ^^^^^^^
../../../dev/flutter/packages/flutter/lib/src/material/elevated_button.dart:151:22: Context: Found this candidate, but the arguments don't match.
  static ButtonStyle styleFrom({
                     ^^^^^^^^^
Target kernel_snapshot failed: Exception


FAILURE: Build failed with an exception.
```

**Comment by joukhar** [comment]
If you want my help you're welcome

**Comment by MakeOpinionGmbH** [comment]
we relased yesterday a update for our sdk at the moment it is under develop in testing. if you want use it use our develop version 

with 

 cpx_research_sdk_flutter:
    git:
      url: https://github.com/MakeOpinionGmbH/cpx-research-SDK-Flutter.git
      path: cpx_research_sdk_flutter
      ref: develop

**Comment by joukhar** [comment]
I have no problem, i can wait, just take your time carefully and update dependencies to avoid vulnerabilities , i need a stable version for my app.

And Thanks for response.

**Comment by crab-by** [comment]
Fixed with Release [0.4.5](https://github.com/MakeOpinionGmbH/cpx-research-SDK-Flutter/releases/tag/0.4.5)
